tox.ini

```diff
@@ -69,6 +69,7 @@ setenv =
     async: PYTEST_PATH = napari
 deps =
     pytest-cov
+    pooch>=1.3.0,!=1.5.0
```

Comment:
shouldn't this be covered by the fact that `testing` appears in the

Reply:
theoretically should but not works :( . I do not have idea why.

Reply:
huh!  strange